src/resources.qrc

```diff
@@ -14,6 +14,9 @@
     <qresource prefix="/translations">
         <file alias="translation_es">translation/translation_es_ES.qm</file>
     </qresource>
+    <qresource prefix="/translations">
```

Comment:
So we'd remove this to not let the translation show up?

Reply:
I think this would work, yes (at least that was the effect when we forgot adding it for Korean, I think?).

So, yes, either that. Or go with the PR as-is and add a reminder to the 3.9.1 checklist to remove that part again if no translations have been integrated at release candidate time.

Reply:
> Or go with the PR as-is and add a reminder to the 3.9.1 checklist to remove that part again if no translations have been integrated at release candidate time

I'd go with that since I understood @comradekingu that he'll start translating right away.